.env.example

```diff
@@ -1,36 +1,13 @@
+# Wordpress Configuration
+WORDPRESS_DB_HOST=mariadb
 WORDPRESS_DB_NAME=wordpress
 WORDPRESS_DB_USER=root
 WORDPRESS_DB_PASSWORD=root
 
-AUTH_KEY=
-SECURE_AUTH_KEY=
-LOGGED_IN_KEY=
-NONCE_KEY=
-AUTH_SALT=
-SECURE_AUTH_SALT=
-LOGGED_IN_SALT=
-NONCE_SALT=
-
-WP_ALLOW_MULTISITE=1
-MULTISITE=1
-SUBDOMAIN_INSTALL=1
-DOMAIN_CURRENT_SITE=localhost
-PATH_CURRENT_SITE=/
-SITE_ID_CURRENT_SITE=1
-BLOG_ID_CURRENT_SITE=1
-
-ONESIGNAL_TOKEN=
-ONESIGNAL_APP=
+# Maria DB Configuraton
+MARIADB_USER=root
+MARIADB_ROOT_PASSWORD=root
+MARIADB_DATABASE=wordpress
 
+# Xdebug Configuration
 XDEBUG_CONFIG="remote_host=172.17.0.1 remote_enable=1"
-
-WORDPRESS_CONFIG_EXTRA="define('WP_MAILHOG_HOST', 'mailhog');
-define( 'SMTP_USER',   'user@example.com' );    // Username to use for SMTP authentication
-define( 'SMTP_PASS',   'smtp password' );       // Password to use for SMTP authentication
-define( 'SMTP_HOST',   'mailhog' );    // The hostname of the mail server
-define( 'SMTP_FROM',   'website@example.com' ); // SMTP From email address
-define( 'SMTP_NAME',   'e.g Website Name' );    // SMTP From name
-define( 'SMTP_PORT',   '1025' );                  // SMTP port number - likely to be 25, 465 or 587
-define( 'SMTP_SECURE', '' );                 // Encryption system to use - ssl or tls
-define( 'SMTP_AUTH',    false );                 // Use SMTP authentication (true|false)
```

Comment:
What we will do to define SMTP settings and don't put this at `wp-config.php` file?